src/test/java/hudson/plugins/sonar/casc/CasCTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * SonarQube Scanner for Jenkins
+ * Copyright (C) 2007-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package hudson.plugins.sonar.casc;
+
+import hudson.plugins.sonar.SonarGlobalConfiguration;
+import hudson.plugins.sonar.SonarInstallation;
+import io.jenkins.plugins.casc.misc.RoundTripAbstractTest;
+import jenkins.model.GlobalConfiguration;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CasCTest extends RoundTripAbstractTest {
+
+    @Override
+    protected void assertConfiguredAsExpected(RestartableJenkinsRule restartableJenkinsRule, String s) {
+        SonarInstallation installation = GlobalConfiguration.all()
+                .get(SonarGlobalConfiguration.class)
+                .getInstallations()[0];
+        assertThat(installation).isNotNull();
+    }
+
+    @Override
+    protected String stringInLogExpected() {
+        return "credentialsId = test-id";
+    }
+}
```

Comment:
I see we forgot to link to our contribution guidelines for this project. Sorry about that.

Could you please reformat this code so it follows our code style (config for Eclipse and IntelliJ available [here](https://github.com/SonarSource/sonar-developer-toolset#eclipse-configuration))? From the looks of it, I think only the indentation should be corrected.

Reply:
> I see we forgot to link to our contribution guidelines for this project. Sorry about that.
> 
> Could you please reformat this code so it follows our code style (config for Eclipse and IntelliJ available [here](https://github.com/SonarSource/sonar-developer-toolset#eclipse-configuration))? From the looks of it, I think only the indentation should be corrected.

Checkstyle? 🙂